spec/index.bs

```diff
@@ -3128,6 +3128,51 @@ dictionary GPUCommandEncoderDescriptor : GPUObjectDescriptorBase {
 };
 </script>
 
+## Pass Encoding ## {#command-encoder-pass-encoding}
+
+### <dfn method for=GPUCommandEncoder>beginRenderPass(descriptor)</dfn> ### {#GPUCommandEncoder-beginRenderPass}
+
+<div algorithm="GPUCommandEncoder.beginRenderPass">
+    <strong>|this|:</strong> of type {{GPUCommandEncoder}}.
+
+    **Arguments:**
+        - {{GPURenderPassDescriptor}} |descriptor|
+
+    **Returns:** {{GPURenderPassEncoder}}
+
+    Begins encoding a render pass described by |descriptor|.
+
+    On the [=Device timeline=], the following steps occur:
+
+        - If the [$GPUCommandEncoder.beginRenderPass/Valid Usage$] rules are met:
+
+            - Set |this|.{{GPUCommandEncoder/[[state]]}} to {{encoder state/encoding a render pass}}.
+
+            - For each |colorAttachment| in |descriptor|.{{GPURenderPassDescriptor/colorAttachments}}:
+
+                - Each texture [=subresource=] seen by |colorAttachment|.{{GPURenderPassColorAttachmentDescriptor/attachment}}
```

Comment:
this shouldn't be "each" if we require there to only be one and exactly one subresource